File: FundamentalsJava/src/NegatiationEx/PrintAndSum.java
package NegatiationEx;

import java.util.Scanner;

public class PrintAndSum {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int startNum = Integer.parseInt(scanner.nextLine());
        int endNum = Integer.parseInt(scanner.nextLine());
        int sum = 0;
        for (int num = startNum; num <= endNum; num++){
            System.out.printf("%d ",num);
            sum += num;

        }
        System.out.printf("%nSum: %d", sum);
    }
}

File: JsFront-end/jsSyntaxFundExercise/printAndSum.js
function sum(startNum, endNum) {
  let sum = 0;
  let result = [];
  for (let i = startNum; i <= endNum; i++) {
    sum += i;
    result.push(i);
  }

  console.log(result.join(" "));
  console.log(`Sum: ${sum}`);
}
